TemplateMethod/src/main/java/devslp/designpatterns/TemplateMethod/SortStrategyExample/IntSortHandler.java

```diff
@@ -0,0 +1,37 @@
+package devslp.designpatterns.TemplateMethod.SortStrategyExample;
+
+/*
+  The IntSortHandler class knows nothing whatever of the BubbleSorter,
+  having no dependency whatever on the bubble sort implementation.
+*/
+public class IntSortHandler implements SortHandler {
+    private int [] array;
```

Comment:
Due to method getArray returning Object[], this declaration should be changed to type Integer[] and cast in line 14 also should be changed to (Integer)